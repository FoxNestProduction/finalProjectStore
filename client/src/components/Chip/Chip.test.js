import React from 'react';
import { render } from '@testing-library/react';
import ColorChips from './Chip';

describe('smoke test', () => {
  test('should Chip render', () => {
    const { asFragment } = render(
    <ColorChips isHealthy={false} isTrending={false} isSupreme={false} />
  );
    expect(asFragment()).toMatchSnapshot();
  });
});
