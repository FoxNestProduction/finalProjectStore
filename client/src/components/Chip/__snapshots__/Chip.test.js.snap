// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test should Chip render 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1ma6dsf"
  >
    <div
      class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-bqy36f-MuiChip-root"
    >
      <span
        class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
      >
        Healthy
      </span>
    </div>
    <div
      class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-17hug3x-MuiChip-root"
    >
      <span
        class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
      >
        Trending
      </span>
    </div>
    <div
      class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-3c6rrk-MuiChip-root"
    >
      <span
        class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
      >
        Supreme
      </span>
    </div>
  </div>
</DocumentFragment>
`;
